### PR TITLE
Added analytics HA docs

### PR DESCRIPTION
**Pull request by ruks**
## Purpose
Added analytics HA docs

### CODE REVIEW

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+`<APIM-ANALYTICS_WORKER_HOME>/conf/worker/deployment.yaml` file.
+
+``` yaml
+siddhi:
+ properties:
+   partitionById: true
+   shardId: wso2-dc
+```
+
+This query captures the information relating to the total request counts for different APIs. Each request represents
```
Comment (fazlan-nazeem) [inline en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md]: Do we need this part?

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+!!! note 
+
+    As explained in above the events are processed in multiple active nodes. Eventhough this is usually a stateful 
+    operation, you can overcome the node-dependent calculations via distributed aggregation. This allows WSO2 API-M to 
+    execute APIM Analytics scripts that depend on incremental distributed aggregation.
+    
+    However, an active-active deployment can affect alerts because alerts also depend on some in-memory stateful 
+    operations such as windows. Due to this, alerts can be generated based on the events received by specific node. Thus 
+    the alerts are node-dependent, and you need to disable them to run scripts with distributed incremental aggregation.
```
Comment (fazlan-nazeem) [inline en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md]: shall we just mention that some alert types only will be affected?  I don't think we need to mention anything about distributed incremental aggregation.

**Comment by Mariangela** [comment]
@ruks is this only applicable for 3.1.0 or it is applicable for 3.0.0 as well

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md (hunk 1 above)
Comment (ruks) [reply en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md]: removed

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md (hunk 2 above)
Comment (ruks) [reply en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md]: fixed

**Comment by Mariangela** [comment]
Reviewed active-active.png https://github.com/wso2/docs-apim/pull/867/commits/0a52a62311611ab85b5b7aae3dd1c648ff1ebde7

**Pull request merged**